### ISSUE TITLE
Will this work with the EU menu?

### ISSUE THREAD

**Issue by roydbt** [Alan-bur/WM4K#1]
In the README it says to download the American menu from dolphin, but all my games and the menu are European. Will this resource pack work with the European menu?

**Comment by Alan-bur** [comment]
Hello there! You have to download the Wii Menu USA Version to make it work. It doesn't work for the Europe version yet. If i see more people collaborate with this and give me some donations, i will for sure make some time to make it compatible with Europe one too. Hope this helps! Remember to follow me on Twitter so you could see the improvements! 🙂
https://x.com/A_Burcet
Also check this group out where we share Frutiger Aero stuff!
https://www.facebook.com/groups/frutigeraeroaesthetic


  *   Alan

________________________________
De: roydbt ***@***.***>
Enviado: jueves, agosto 15, 2024 6:51 AM
Para: Alan-bur/Wii-4K ***@***.***>
Cc: Subscribed ***@***.***>
Asunto: [Alan-bur/Wii-4K] Will this work with the EU menu? (Issue #1)


In the README it says to download the American menu from dolphin, but all my games and the menu are European. Will this resource pack work with the European menu?

**Comment by roydbt** [comment]
Do you happen to know what changes need to be made to support the PAL region?

**Comment by Alan-bur** [comment]
Yes, you gotta first install the Europe menu, then go into the textures dump folder and start upscaling all of the textures you see in that folder into 4K. If you made everysingle one of those textures, make a pull request and i would be glad to put it inside the project. 😅👌
________________________________
De: roydbt ***@***.***>
Enviado: viernes, agosto 16, 2024 1:27 PM
Para: Alan-bur/Wii-4K ***@***.***>
Cc: Alan ***@***.***>; Comment ***@***.***>
Asunto: Re: [Alan-bur/Wii-4K] Will this work with the EU menu? (Issue #1)


Do you happen to know what changes need to be made to support the PAL region?

**Comment by louisefindlay23** [comment]
[Dumped EU Menu Textures](https://github.com/user-attachments/files/16680565/0000000100000002.zip)

Some of the textures seem to be shared because I tried enabling the texture pack on both a US and Europe Wii Menu and when I enabled the texture pack for the EU menu (I toggled Load Custom Textures off and on to tell the difference), the upscaled channel images remained.

**Comment by Alan-bur** [comment]
Yes, some of them are shared. That's right.
________________________________
De: Louise ***@***.***>
Enviado: martes, agosto 20, 2024 3:57 PM
Para: Alan-bur/Wii-4K ***@***.***>
Cc: Alan ***@***.***>; Comment ***@***.***>
Asunto: Re: [Alan-bur/Wii-4K] Will this work with the EU menu? (Issue #1)


Dumped EU Menu Textures<https://github.com/user-attachments/files/16680565/0000000100000002.zip>

Some of the textures seem to be shared because I tried enabling the texture pack on both a US and Europe Wii Menu and when I enabled the texture pack for the EU menu (I toggled Load Custom Textures off and on to tell the difference), the upscaled channel images remained.

**Comment by Alan-bur** [comment]
This will probably be fixed on future updates as i'm currently working on other projects. Remember to join the discord server in case you would like to see more about what i create and what i do. Thank you!
